client/src/components/route/PrivateRoute.js

```diff
@@ -5,48 +5,54 @@ import { Switch, Redirect } from 'react-router-dom';
 import React, { useEffect } from 'react';
 import { useDispatch } from 'react-redux';
 import Cookies from 'js-cookie';
-// Pages 
+// Pages
 import Dashboard from '../dashboard/Dashboard';
 import ProfileInfoEdit from '../profileInfoEdit/ProfileInfoEdit';
 import PublicationPage from '../publications/PublicationPage';
 import TeamPage from '../teamPage/TeamPage';
+import ClientHomeEditor from '../clientHomeEditor/ClientHomeEditor';
 
 // Layout
 import DashboardLayoutRoute from '../layouts/dashboardLayout/DashboardLayoutRoute';
 import { logout } from '../../actions/auth';
 
 const PrivateRoute = () => {
-  const dispatch = useDispatch()
+  const dispatch = useDispatch();
   const signInCookie = Cookies.get('isLogin') ? true : false;
 
   useEffect(() => {
     // after a user signIn, we need to keep track of the signInCookie as it might be removed after the tokens expire
-    // if that the case, a sign out action need to be dispatched 
-    if(!signInCookie){
+    // if that the case, a sign out action need to be dispatched
+    if (!signInCookie) {
       setTimeout(() => {
-        dispatch(logout())
-      }, 3000)
+        dispatch(logout());
+      }, 3000);
     }
-  }, [dispatch, signInCookie])
+  }, [dispatch, signInCookie]);
 
-    return (
-        <Switch>
-          <DashboardLayoutRoute path="/dashboard" exact component={Dashboard} />
-          <DashboardLayoutRoute
-            path="/dashboard/profile"
-            exact
-            component={ProfileInfoEdit}
-          />
-          <DashboardLayoutRoute
-            path={`/publications`}
-            exact
-            component={PublicationPage}
-          />
-          <DashboardLayoutRoute path="/team" exact component={TeamPage} />
-          {/*  If login, any other route not stated above will be redirect dashbroad page */}
-          <Redirect to="/dashboard"/> 
-        </Switch> 
-    );
-  };
+  return (
+    <Switch>
+      <DashboardLayoutRoute path="/dashboard" exact component={Dashboard} />
+      <DashboardLayoutRoute
+        path="/dashboard/profile"
+        exact
+        component={ProfileInfoEdit}
+      />
+      <DashboardLayoutRoute
+        path={`/publications`}
```

Comment:
Don't need to use ``/publications`` template strings here when normal string can be used

Reply:
changed